pkg/tracegen/util.go

```diff
@@ -25,9 +25,9 @@ import (
 	"go.elastic.co/apm/v2"
 )
 
-// NewRandomTraceID returns randomly generated apm.TraceID
+// newRandomTraceID returns randomly generated apm.TraceID
 // which is also compatible with otel's trace.TraceID
-func NewRandomTraceID() apm.TraceID {
+func newRandomTraceID() apm.TraceID {
 	var traceID apm.TraceID
 	r := rand.New(rand.NewSource(time.Now().UnixNano()))
 	binary.LittleEndian.PutUint64(traceID[:8], r.Uint64())
```

Comment:
I think it'd be better to seed the package random generator and use that one

```go
func init() {
	rand.Seed(time.Now().UnixNano())
}

// RandomTraceID returns randomly generated apm.TraceID
// which is also compatible with otel's trace.TraceID
func RandomTraceID() apm.TraceID {
	var traceID apm.TraceID
	binary.LittleEndian.PutUint64(traceID[:8], rand.Uint64())
	binary.LittleEndian.PutUint64(traceID[8:], rand.Uint64())
	return traceID
}

```

Reply:
Agreed, but without the explicit seed as long as we're on Go 1.20+. `math/rand.Seed` was deprecated in Go 1.20, and the top-level RNG is automatically seeded at program init time.

Reply:
oh I see. Thanks for noticing!